mappings/net/minecraft/loot/LootPool.mapping

```diff
@@ -34,6 +34,8 @@ CLASS net/minecraft/class_55 net/minecraft/loot/LootPool
 		METHOD method_352 rolls (Lnet/minecraft/class_5658;)Lnet/minecraft/class_55$class_56;
 			ARG 1 rolls
 		METHOD method_355 build ()Lnet/minecraft/class_55;
+		METHOD method_35509 bonusRolls (Lnet/minecraft/class_5658;)Lnet/minecraft/class_55$class_56;
```

Comment:
Ooh, nice that this is now in the builder too. Need to add it to my loot v2 API PR.